### PR TITLE
Use base image to inherit python setup

### PR DESCRIPTION
**Pull request by akinnane**

### CODE REVIEW

**Comment by akinnane** [comment]
The error from the pipeline 
Screenshot from 2021-02-11 17-55-42

**Comment by OllieJC** [comment]
Some links to make current usage work: https://github.com/alphagov/centralised-security-logging-service/blob/c6a6a4334061b2973e554adc2942063edfd3ea5b/config/concourse/pipeline/build-csls-pipeline.yml#L414
``` hcl
FROM gdscyber/cyber-security-concourse-base-image

RUN apt-get update -y && \
        apt-get install -y nodejs npm && \
        apt-get clean

RUN npm install --ignore-scripts artillery && \
    npm install --ignore-scripts artillery-engine-kinesis

RUN mkdir -p /node_modules/artillery/bin && \
    ln -s /tmp/node_modules/artillery/bin/artillery \
        /node_modules/artillery/bin/artillery && \
    ln -s /tmp/node_modules/artillery/bin/artillery \
        /usr/bin/artillery

VOLUME /artillery
WORKDIR /artillery

ENTRYPOINT ["/bin/bash"]
```

File: load_testing/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-RUN apt-get update -y
-RUN apt-get install -y python3.7 python3-distutils python3-pip
-RUN apt-get install -y build-essential libssl-dev libffi-dev python3-dev
-RUN apt-get install -y python3-venv
-RUN apt-get install -y awscli jq curl
-RUN apt-get install -y nodejs npm
-RUN npm install --ignore-scripts artillery && npm install --ignore-scripts artillery-engine-kinesis
+RUN npm install -g \
+        artillery \
+        artillery-engine-kinesis
```
Comment (OllieJC) [inline load_testing/Dockerfile:9]:
```suggestion
        artillery-engine-kinesis
        
RUN mkdir -p /node_modules/artillery/bin && \
    ln -s /usr/local/bin/artillery \
        /node_modules/artillery/bin/artillery
```

**Review by OllieJC — APPROVED** [review]
Works on my machine ™

**Pull request merged**